client/src/components/ProductDetails/sampleData/productListDummyData.js

```diff
@@ -0,0 +1,129 @@
+
```

Comment:
ProductDetails folder also has a productListDummyData file that is empty. Did you mean to leave it there?

Reply:
That is weird. The file is not empty in my branch

<img width="790" alt="Screen Shot 2021-12-27 at 10 35 47 PM" src="https://user-images.githubusercontent.com/59572783/147525363-5d041609-4e4e-4d76-a458-9fa52ff9f478.png">

Reply:
The path for the empty file is => client/src/components/ProductDetails/productListDummyData.js
The file you're showing is inside sampleData. It seems right to me. 
I couldn't comment on the correct file because it's empty.

<img width="779" alt="Screen Shot 2021-12-28 at 8 14 09 AM" src="https://user-images.githubusercontent.com/75865828/147579141-4aa458e0-f9ad-479f-aee2-ea7db668c1b5.png">